example/src/androidTestDebug/java/com/stripe/example/activity/PaymentSessionActivityTest.kt

```diff
@@ -49,8 +49,10 @@ class PaymentSessionActivityTest {
             .perform(click())
 
         // on PaymentMethodsActivity
-        Espresso.onView(ViewMatchers.withId(R.id.recycler)).perform(
-            RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(0, click())
-        )
+
```

Comment:
What is the final action on this comment?

Reply:
Good question, I'll just remove this for now.